frontend/src/components/v-ramp.vue

```diff
@@ -2,20 +2,22 @@
 .ramp
   template(v-if="tframe === 'commit'")
     template(v-for="(slice, j) in user.commits")
-      a.ramp__slice(
-        draggable="false",
-        v-on:click="rampClick",
-        v-for="(commit, k) in slice.commitResults.filter(commitResult => commitResult.insertions > 0)",
-        v-bind:href="getLink(commit)", target="_blank",
-        v-bind:title="getContributionMessage(slice, commit)",
-        v-bind:class="'ramp__slice--color' + getSliceColor(slice.date)",
-        v-bind:style="{\
-          zIndex: user.commits.length - j,\
-          borderLeftWidth: getWidth(commit) + 'em',\
-          right: ((getSlicePos(slice.date)\
-            + (getCommitPos(k, slice.commitResults.length))) * 100) + '%'\
-          }"
-      )
+      template(v-for="(commit, k) in slice.commitResults.filter(commitResult => commitResult.insertions > 0)")
+        a.ramp__slice(
+          draggable="false",
+          v-on:click="rampClick",
+          v-bind:href="getLink(commit)", target="_blank",
+          v-if="!isBrokenLink(getLink(commit))",
```

Comment:
Can I check: Does this mean that if the repository is not supported, then there will not be any ramps shown on the dashboard for that repository?

Reply:
Oh yeah, if the repository is not supported, the remote link would be undefined. The link to blame view and link to history will also not shown. So I can remove the conditional statement for ramp, blame view and history component?

Reply:
@dcshzj Thank you for the review! I found a bug in my code that hide the broken link ramp, I have fixed it and the ramps with broken link are visible now. Could you have a look?